backend/locales/zh-CN.json

```diff
@@ -0,0 +1,183 @@
+    "decky_update_available": "新版本 {{tag_name}} 可用！",
+    "error": "Error",
+    "plugin_error_uninstall": "如果你想卸载插件请点击 Decky 菜单中的 {{-icon}} 图标",
+    "plugin_load_error": {
+      "message": "加载插件 {{name}} 错误",
+      "toast": "加载插件 {{name}} 发生了错误"
+    },
+    "plugin_uninstall": {
+      "button": "卸载",
+      "desc": "你确定要卸载 {{name}} 吗?",
+      "title": "卸载 {{name}}"
+    },
+    "plugin_update_one": "1 个插件有更新！",
+    "plugin_update_other": "{{count}} 个插件有更新！"
+  },
+  "RemoteDebugging": {
+    "remote_cef": {
+      "desc": "允许你网络中的任何人无需身份验证即可访问CEF调试器",
+      "label": "允许远程访问CEF调试"
+    }
+  },
+  "SettingsDeveloperIndex": {
+    "header_other": "其他",
+    "react_devtools": {
+      "desc": "允许连接到运行着 React DevTools 的计算机，更改此设置将重新加载Steam，请在启用前设置IP地址",
+      "ip_label": "IP",
+      "label": "启用 React DevTools"
+    },
+    "third_party_plugins": {
+      "button_install": "安装",
+      "button_zip": "浏览文件",
+      "header": "第三方插件",
+      "label_desc": "URL",
+      "label_url": "从 URL 安装插件",
+      "label_zip": "从 ZIP 压缩文件安装插件"
+    },
+    "toast_zip": {
+      "body": "安装失败！只有 ZIP 格式的插件被支持",
+      "title": "Decky"
+    },
+    "valve_internal": {
+      "desc1": "启用 Valve 内部开发者菜单",
+      "desc2": "除非你知道你在干什么，否则请不要修改此菜单中的任何内容",
+      "label": "启用 Valve 内部开发者"
+    }
+  },
+  "SettingsGeneralIndex": {
+    "about": {
+      "decky_version": "Decky 版本",
+      "header": "关于"
+    },
+    "beta": {
+      "header": "测试版"
+    },
+    "developer_mode": {
+      "desc": "启用 Decky 的开发者测试",
+      "label": "开发者模式"
+    },
+    "other": {
+      "header": "其他"
+    },
+    "updates": {
+      "header": "更新"
+    }
+  },
+  "SettingsIndex": {
+    "developer_title": "开发者",
+    "general_title": "通用",
+    "navbar_settings": "Decky 设置",
+    "plugins_title": "插件"
+  },
+  "Store": {
+    "store_contrib": {
+      "desc": "如果你想要提交你的插件到 Decky 插件商店，请访问 GitHub 上的 SteamDeckHomebrew/decky-plugin-template 存储库，关于开发和分发的相关信息，请查看 README 文件",
+      "label": "贡献"
+    },
+    "store_filter": {
+      "label": "过滤器",
+      "label_def": "全部"
+    },
+    "store_search": {
+      "label": "搜索"
+    },
+    "store_sort": {
+      "label": "排序",
+      "label_def": "最后更新 (最新)"
+    },
+    "store_source": {
+      "desc": "所有插件的源代码都可以在 GitHub 上的 SteamDeckHomebrew/decky-plugin-database 存储库中获得",
+      "label": "源代码"
+    },
+    "store_tabs": {
+      "about": "关于",
+      "alph_asce": "字母排序 (Z 到 A)",
+      "alph_desc": "字母排序 (A 到 Z)",
+      "title": "浏览"
+    },
+    "store_testing_cta": "请考虑测试新插件以帮助 Decky Loader 团队！"
+  },
+  "StoreSelect": {
+    "custom_store": {
+      "label": "自定义商店",
+      "url_label": "URL"
+    },
+    "store_channel": {
+      "custom": "自定义",
+      "default": "默认",
+      "label": "商店通道",
+      "testing": "测试"
+    }
+  },
+  "Updater": {
+    "decky_updates": "Decky 更新",
+    "no_patch_notes_desc": "此版本没有补丁说明",
+    "patch_notes_desc": "补丁说明",
+    "updates": {
+      "check_button": "检查更新",
+      "checking": "'检查中",
```

Comment:
The initial ' after " was a typo in the english translation, please remove that.